modules/strelka/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    strelka:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            sample_bam: null  # UPDATE
+            sample_bai: null  # UPDATE
+            candidate_indel: null  # UPDATE
+
+        scratch_subdirectories: []
+
+        options:
+            configure:
+                mrna: "--rna"
+                capture: "--exome"
+            strelka: "-m local"
+
+        conda_envs:
+            strelka: "{MODSDIR}/envs/strelka-2.9.10.yaml"
+            tabix: "{MODSDIR}/envs/tabix-0.2.6.yaml"
+            
+        threads:
+            strelka: 4
+
+        mem_mb:
+            strelka: 2000
+
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            capture:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
```

Comment:
A few of your files don't end with a newline (indicated by a red `–` in GitHub). You can avoid this in VS Code with `"files.insertFinalNewline": true`.

Reply:
thank you!